.circleci/config/jobs/@jobs.yml

```diff
@@ -235,14 +235,17 @@ eks_integration_tests:
     - checkout
     - install_python_requests
     - setup_node16
+    - aws-cli/install:
+        override-installed: true
     - run:
         name: Create temp dir for logs
         command: mkdir -p /tmp/logs/test/integration/eks
     - run:
         name: Integration tests EKS
         command: |
+          npm install &&
           export KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG=$(./scripts/circleci-jobs/setup-integration-tests.py)
-          .circleci/do-exclusively --branch staging --job ${CIRCLE_JOB} npm run test:integration:eks:yaml
```

Comment:
Removed `do-exclusively` as we control flow with the branches, so no need to do that

Reply:
How do we control it with the branches?

Reply:
https://github.com/snyk/kubernetes-monitor/blob/staging/.circleci/config/%40config.yml#L71

Reply:
https://snyk.slack.com/archives/C022DN6CMHS/p1627308110027800?thread_ts=1627307326.026800&cid=C022DN6CMHS